build.sbt

```diff
@@ -138,7 +139,7 @@ lazy val natchezHttp4s = projectMatrix
       name := "natchez-extras-http4s-stable",
       libraryDependencies ++= Seq(
         "org.http4s" %% "http4s-dsl" % http4sStableVersion,
-        "org.http4s" %% "http4s-client" % http4sStableVersion
+        "org.http4s" %% "http4s-client" % blazeStableVersion
```

Comment:
See above.

Reply:
yep, reverted to the original version, thanks!